AccountRecordsBasedOnBillingState.cls

```diff
@@ -0,0 +1,19 @@
+public with sharing class AccountRecordsBasedOnBillingState{
+
+  /**Method to display account records where billing states are not maharashtra and kerala**/
+  public void accountRecords()
+   {
+     List<account> listAccount = [Select name,phone,BillingState 
+                                 FROM Account 
+                                 WHERE BillingState 
+                                 NOT IN('MAHARASHTRA','KERALA') 
```

Comment:
@Manisha7992 Instead of Using state list directly in Query you should define list variable of type `string` and then add required states into that list and then use that variable in query. This will make your query more dynamic so when tommorw you want to add any state to query you can simply add it into list. no need to change query.